### ISSUE TITLE
[$30]Inconsistent landmarks across pages.

### ISSUE THREAD

**Issue by veshu** [topcoder-platform/community-app#3234]
**Describe the bug**

Inconsistent landmarks across pages.

**To Reproduce**

1. Go to the sample challenge page
2. Ensure that the screen reader is switched on. 
3. Press the letter 'r' a couple of times to navigate to the various landmarks on the page. 
4. Note that there is no 'banner' landmark for the header, 'main landmark' for the main content of the page, and 'contentinfo landmark' for the footer of the page. 
5. Also note that this issue can be found on the other pages too. The landmarks are inconsistent."

**Expected behavior**

The content on the page should be placed inside the relevant landmark roles for screen reader users to skip to the content in the respective landmark he/she intends to read. The banner landmark should contain the header of the page. The main landmark should be wrapped around the main content of the page. The contentinfo should be used to contain the footer of the page. Navigation, search etc can be used additionally.

**Actual behavior**

There are no banner, main and contentinfo landmarks on the page.

non-active and active login button do not comply with WCAG.  Make it the same color scheme as registration page buttons "

**Additional context**

Type: Landmarks

Page: https://www.topcoder.com/challenges/30095347

Status: Partially fixed on a few pages. Correctly fixed on the sample challenge page.

**Comment by crazyk07** [comment]
Contest https://www.topcoder.com/challenges/30099914 has been created for this ticket.<br/><br/>```This is an automated message for crazyk via Topcoder X```

**Comment by crazyk07** [comment]
Contest https://www.topcoder.com/challenges/30099914 has been updated - the new changes has been updated for this ticket.<br/><br/>```This is an automated message for crazyk via Topcoder X```

**Comment by crazyk07** [comment]
Contest https://www.topcoder.com/challenges/30099914 has been updated - it has been assigned to rohitgupta_.<br/><br/>```This is an automated message for crazyk via Topcoder X```

**Comment by r0hit-gupta** [comment]
@veshu can you please provide with the links where the landmarks are not present?

**Comment by veshu** [comment]
@r0hit-gupta https://www.w3.org/TR/WCAG20-TECHS/ARIA11.html this is the guidelines for this.

We need to separate the landmarks so user can outline the regions on challenge detail page. 

Note: This behavior should also be fixed on other pages (listing, login/register, submit page)

**Comment by r0hit-gupta** [comment]
PRs at https://github.com/appirio-tech/accounts-app/pull/303 & https://github.com/topcoder-platform/community-app/pull/3255

**Comment by tosha5252** [comment]
Also note that this issue can be found on the other pages too. The landmarks are inconsistent."

Status: Partially fixed on a few pages. Correctly fixed on the sample challenge page.

@sushilshinde what pages does this issue cover?

**Comment by veshu** [comment]
@tosha5252 Current scope pages are challenge listing, challenge detail and submit page,
Register page, login page.

**Comment by drasticdpk** [comment]
@r0hit-gupta Can we make this page consistent , since screen reader does not read **NOT A MEMBER YET**  on login page but reads **HAVE AN ACCOUNT** on registration page.
Here is the reference :
![image](https://user-images.githubusercontent.com/25297859/64435126-a66ea980-d0e1-11e9-8c70-e43a38c717c6.png)
![image](https://user-images.githubusercontent.com/25297859/64435216-cbfbb300-d0e1-11e9-9835-37062cee497a.png)

**Comment by r0hit-gupta** [comment]
@drasticdpk PR at https://github.com/appirio-tech/accounts-app/pull/311. This should fix it.

**Comment by crazyk07** [comment]
Closing ticket due to shift in priority, will not be able to complete but will still pay for effort rendered.

**Comment by crazyk07** [comment]
Payment task has been updated: https://software.topcoder.com/review/actions/ViewProjectDetails?pid=30099914<br/><br/>```This is an automated message for crazyk via Topcoder X```
